system/jlib/jlog.hpp

```diff
@@ -236,6 +236,16 @@ typedef unsigned LogMsgDetail;
 #define DefaultDetail   100
 #define TopDetail (LogMsgDetail)-1
 
+constexpr int AllMsgThreshold         = 1;
+constexpr int FatalMsgThreshold       = 1;
+constexpr int ErrMsgThreshold         = 20-1;
+constexpr int WarnMsgThreshold        = 30-1;
```

Comment:
does this suggest 2-19 are categorized as error messages?
20-29 are categorized as warning messages?
For clarity would be good to add a comment saying these are ranges, x->y